LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
EzINI is actually LGPL v3, not GPL V3, so this file should be changed. See `src/COPYING.LESSER`.

Reply:
Just saw this and hummm the original author provided both licenses... confusing, but ok let me fix that!

Reply:
That is probably because LGPL references GPL. As of v3 LGPL is an extension of GPL, I think.